README.md

```diff
@@ -122,6 +122,10 @@ Where do I set this?
   - `waitingSubtitle`. Example value: "Your clinician is joining the meeting".
   - `logoUrl`. Example value: "https://your_cdn/logo.png".
 
+### End to End tests
```

Comment:
Minor suggestion to move this section after Build and Run and before Environment Variables